CMakeLists.txt

```diff
@@ -67,13 +68,26 @@ include_directories(${PROJECT_SOURCE_DIR}/third_party/rvi_lib/include)
 include_directories(${LIBOSTREE_INCLUDE_DIRS})
 include_directories(${LIBDBUS_INCLUDE_DIRS})
 
-set_source_files_properties(${PROJECT_SOURCE_DIR}/third_party/jsoncpp/jsoncpp.cpp PROPERTIES COMPILE_FLAGS -w)
-
 if(BUILD_GENIVI)
-    set_property(GLOBAL PROPERTY CXX_STANDARD 11)
```

Comment:
Cmake doesn't have a specific c++03 option. The code still compiles if c++98 is used without -Wpedantic. That checks for several things that cause problems.

I didn't realize we might be restricted to C++03. In fact, there already was a specific reference to building with C++11, and it was set as a global property.

And by the way, I've now incorporated scoped_arrays.